stdlib/public/core/LifetimeManager.swift

```diff
@@ -166,11 +166,22 @@ func __abi_withUnsafePointer<T, Result>(
 ///     `withUnsafeMutablePointer(to:_:)` instead.
 /// - Returns: The return value, if any, of the `body` closure.
 @inlinable
-public func withUnsafePointer<T, Result>(
+@_alwaysEmitIntoClient
+public func withUnsafePointer<T, Result, E: Error>(
+  to value: inout T,
+  _ body: (UnsafePointer<T>) throws(E) -> Result
+) throws(E) -> Result {
+  try body(UnsafePointer<T>(Builtin.addressof(&value)))
+}
+
+/// ABI: Historical withUnsafePointer(to:_:) rethrows,
+/// expressed as "throws", which is ABI-compatible with "rethrows".
+@_silgen_name("$ss17withUnsafePointer2to_q_xz_q_SPyxGKXEtKr0_lF")
+@usableFromInline
+func __abi_se0413_withUnsafePointer<T, Result>(
```

Comment:
Since SE-0333, I've included the proposal number the ABI-compatibility symbols being replaced (search for "se0"). This has not been the case so far in the SE-0413 process. Should we start?